utils/weight-diff/src/main.rs

```diff
@@ -437,6 +454,37 @@ fn main() {
                 }
             });
 
+            /* some extra weights: we do not bother with parsing core-processor files, it's too complicated and dumps too much stuff */
+            declarations.push(quote! {
+                pub struct ExtraWeights {
+                    pub rent_waitlist: u64,
+                    pub rent_dispatch_stash: u64,
+                    pub rent_reservation: u64,
+                    pub read: u64,
+                    pub write: u64,
+                }
+            });
+
+            let waitlist = dump.waitlist;
+            let dispatch_stash = dump.dispatch_stash;
+            let reservation = dump.reservation;
+            let read = dump.read;
+            let write = dump.write;
+
+            declarations.push(quote! {
```

Comment:
it seems to be done automatically through struct visitor. you only need to add structure name 2 times (search in code for `"InstantiationWeights"`)

Reply:
https://github.com/gear-tech/gear/blob/9482418e30c0b5fdd6644e3448271e1e3fe75407/utils/weight-diff/src/main.rs#L422-L430

Reply:
I've added the fields to pallet gear Schedule so I assume modifications for weight-diff utility are not required anymore